### DISCUSSION TITLE
SP-API Submitting/Checking Transparency Serial Numbers

### DISCUSSION THREAD

**Discussion by dathat** [amzn/selling-partner-api-models#3745] [General]
We began doing merchant fulfillment on certain items that require transparency codes to be submitted to Amazon. We tried submitting the serial numbers in Seller Central and purchasing the labels using MerchantFulfillmentAPI (without entering any transparency information) as we previously fulfilled orders. However, the serial numbers entered on Seller Central were apparently removed when the label was created, because we received a non-compliance notice about missing serials.

Today I integrated transparency code submission into the createShipment call in MerchantFulfillmentAPI, but I do not have any way to verify the data is submitted correctly. I cannot find anywhere on Seller Central or in SP-API calls where the stored transparency code is visible (they do not show anything even when submitted on Seller Central).

How do other sellers verify transparency codes are attached to order items correctly, or add them to already shipped orders? Trying to avoid any more non-compliance issues.

**Comment by dathat** [comment]
Bump, still cannot verify serial number for MFN transparency order items and curious how others handle it.

https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference
OrderItem returns a SerialNumbers field, but only for FBA orders: 
> A list of serial numbers for electronic products that are shipped to customers. Returned for FBA orders only.

I tried downloading an order report through Seller Central with the transparency and serial number columns added. The rows with "true" for "is-transparency" column have a blank "serial-numbers" column. I am not sure if they were submitted incorrectly, or if the reports only return serial numbers for FBA orders like the API. 

We have very few transparency orders and none through FBA, so testing is complicated.

**Comment by dathat** [comment]
Bumping again, since it is still not clear how to check if transparency codes are recorded correctly. 

I noticed "serial numbers" cannot be submitted via API, only "transparency codes" (according to this: https://sellercentral.amazon.com/help/hub/reference/202008510). That may have been part of our problem.

Tried purchasing labels through Seller Central since that seems to be the only option, but still cannot see whether they recorded. Guess we'll find out in a couple days if our account gets flagged.

Is nobody else experiencing issues around this?

**Comment by petsandall-hub** [comment]
Did you ever figure this out? Same issue here.

**Reply by dathat** [reply]
Nope, never made any progress on it.
